README.md

```diff
@@ -29,9 +29,9 @@ This repository contains:
 
 1. Download the development toolchain (gcc and make) and serial interface software
    
-   For Ubuntu
+   For Ubuntu (18.04 preferred)
```

Comment:
can you rebase off latest master, so the diff is correct

Reply:
I'm getting a bunch of merge conflicts trying to rebase